dev-requirements.txt

```diff
@@ -16,3 +16,4 @@ coveralls
 mdanalysis
 black
 znvis
+ase
```

Comment:
We could also remove a piece of information from the already existing files, I guess the MD is pretty fast but then we don't need such a big package in the dev-req

Reply:
I would not worry about the dev-req size. We also have mdanalysis in there for example.
This way we also catch changes in ASE and we can use it to produce other files that we might want to be able to read.
ASE is about 4 MB compressed.

Reply:
What we could do, is put the generation of the files into DataHub / DataHub CI if you feel very uncomfortable with having it here. But this makes maintaining it more difficult.

Reply:
nope I am happy with that. Sorry I did not read the previous review comments before making my review so did not see that I asked opposing points.